packages/xstate-wallet/src/actions.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-restricted-imports
+export {escalate} from 'xstate/lib/actions';
```

Comment:
Why does this warrant its own file?

Reply:
`escalate` is not exported from the `xstate` module. I thought it would be better to handle that issue once instead of on a per-use basis.